ui/client/components/graph/node-modal/editors/expression/EditableExpression.js

```diff
@@ -0,0 +1,76 @@
+import RawEditor from "./RawEditor"
+import BoolEditor from "./BoolEditor"
+import React from "react"
+import ExpressionWithFixedValues from "./ExpressionWithFixedValues"
+import {parseableBoolean} from "./ExpressionParser"
+
+export default class EditableExpression extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      displayRawEditor: true
+    }
+  }
+
+  render() {
+    const {fieldType, expressionObj, rowClassName, valueClassName, showSwitch} = this.props
+    const editorName = resolveEditorName(fieldType, expressionObj, this.state.displayRawEditor)
+    const Editor = editorTypes[editorName]
```

Comment:
What will happen when editorName not exists in editorTypes?

Reply:
resolveEditorName return by default rawEditor so this situation should not happen